xskillscore/core/probabilistic.py

```diff
@@ -555,16 +556,20 @@ def rps(
           for ``category_edges`` for forecasts. Use this if your category edges vary
           across dimensions of forecasts and observations, and are different for each.
 
-        - None: expect than observations and forecasts are already CDFs containing
-          ``category_edge`` dimension. Use this if your category edges vary across
-          dimensions of forecasts and observations, and are different for each.
+        - None: expect than observations and forecasts are already cumulative ``cdf``
+          or probability ``pdf`` distribution functions specified by ``category_dist``
+          containing ``category`` dimension. Use this if your category edges vary across
```

Comment:
I don't understand the second sentence here. What about "Use this if you have already calculated the cumulative/probability density functions from your observations and forecasts and you wish to use these to calculate the rps."